packages/paste-core/components/button-group/src/ButtonGroup.tsx

```diff
@@ -79,6 +79,8 @@ export const ButtonGroup = React.forwardRef<HTMLDivElement, ButtonGroupProps>(
         ref={ref}
         element={element}
         display="inline-flex"
+        flexWrap={attached ? "nowrap" : "wrap"}
+        rowGap="space40"
```

Comment:
Does this add any additional space below the ButtonGroup is it's attached too? Might impact spacings on rollout

Reply:
rowGap should only apply spacing in between flex rows! Not above or below the flex object itself.
<img width="641" alt="image" src="https://github.com/twilio-labs/paste/assets/75342690/b3b3e01b-c876-4688-b947-2b5d888f2ae5">